tests/kernel/mem_protect/mem_protect/src/mem_domain.c

```diff
@@ -331,3 +331,34 @@ void test_mem_domain_migration(void)
 
 	k_thread_join(&child_thread, K_FOREVER);
 }
+
+
+K_MEM_PARTITION_DEFINE(app_part, ro_buf, sizeof(ro_buf),
+		       K_MEM_PARTITION_P_RW_U_RW);
+
+/**
+ * @brief Test system assert when new partition overlaps existing partition
+ *
+ * When adding the new partition to a memory domain the system must assert that
+ * it does not overlap with any other existing partitions in the domain.
+ * Test to add new partition which has same start address as an existing one,
+ * after that must happen an assertion error indicating that new partition
+ * overlaps existing one.
+ *
+ * @ingroup kernel_memprotect_tests
+ *
+ * @see k_mem_domain_add_partition()
+ */
```

Comment:
Could you take look at the doxygen comments in #29912 and update doxygen comments here? We need that kind of doxygen comments to generate testing specification.

Reply:
Replaced doxygen tag.